worldeditadditions/utils/parse/key_instance.lua

```diff
@@ -0,0 +1,69 @@
+--- A container for transmitting axis, sign pairs within parsing functions.
+-- @class
+local key_instance = {}
+key_instance.__index = key_instance
+key_instance.__name = "Key Instance"
+
+--- Creates a new Key Instance.
```

Comment:
I'm not totally sure what a key_instance is just by looking at this, so a second description line would be a good idea I think. Example:
```suggestion
--- Creates a new Key Instance.
-- Your second description line goes here.
```

Reply:
Oh, good idea.

Reply:
Done.